openmrs/apps/reports/PECG_REPORT/sql/indicator3_ARV_new_treatment.sql

```diff
@@ -139,7 +139,17 @@
             join
                 patient_program_attribute ppa 
                     on ppa.patient_program_id=pp.patient_program_id     
-                    and cast(ppa.value_reference AS DATE) between '#startDate#'AND '#endDate#'    
+                    and cast(ppa.value_reference AS DATE) between '#startDate#'AND '#endDate#'   
+            join
+                  obs o
+                  on o.person_id=p.person_id and o.voided=0
+                  and cast(o.obs_datetime AS DATE) between '#startDate#'AND '#endDate#'   
+            join 
+                  concept c
+                  on c.concept_id=o.concept_id and c.retired=0
+            join concept_name cn 
+                  on cn.concept_id=c.concept_id and cn.voided=0 and concept_name_type ='FULLY_SPECIFIED' 
```

Comment:
@IipperMangesh I haven't been able to dispense yet (the D is not appearing in my environment), busy looking into it.

Reply:
@IipperMangesh This won't work when the locale is French, we should add the Dispensed translation and add it to the query? cn.name ='Dispensed' or cn.name ='Dispensed FR'?